test/test_transmon_expanded.py

```diff
@@ -193,22 +193,22 @@ def test_signals():
     np.testing.assert_allclose(
         actual=test_data["signal_q1"]["ts"],
         desired=data["signal_q1"]["ts"],
-        atol=1e-11 * np.max(data["signal_q1"]["ts"]),
+        atol=1e-8,
```

Comment:
This `atol` value is the same in multiple places in this file. Can we make it a global constant at the top of the file/section?